custom-words.txt

```diff
@@ -1749,3 +1749,9 @@ cloudManifestFiles
 highestcriticalimpact
 apiserver
 kubeadmin
+userprincipalname
+sessionstate
+registation
+sessionhosts
+hostpool
+reseting
```

Comment:
not necessary, correct spelling is 'resetting'

Reply:
ok